pkg/actions/diagnostics.go

```diff
@@ -155,28 +155,30 @@ func dgRemoteCommand(c *cli.Context) {
 		errDG("kube_client_error", "Unable to retrieve Kubernetes clientset: "+err.Error())
 		os.Exit(1)
 	}
-	nameSpacePods, nspErr := clientset.CoreV1().Pods(kubeNameSpace).List(metav1.ListOptions{})
+	cwBasePods, nspErr := clientset.CoreV1().Pods(kubeNameSpace).List(metav1.ListOptions{LabelSelector: "codewindWorkspace=" + clientID})
```

Comment:
Generally in cwctl and the operator we've been using clientsets and workspaceIDs,  here we're calling the workspaceID a clientID.  Could we rename this to be workspaceID so that it matches our convention ?

Reply:
Sure thing, done!